src/Mimi.jl

```diff
@@ -7,18 +7,21 @@ using NamedArrays
 
 export
     ComponentState, run_timestep, run, @defcomp, Model, setindex, addcomponent, setparameter,
-    connectparameter, setleftoverparameters, getvariable, adder, MarginalModel, getindex,
-    getdataframe, components, variables, getvpd, unitcheck, set_external_parameter, plot, getindexcount,
-    getindexvalues, getindexlabels, delete!, get_unconnected_parameters, Timestep, isfirsttimestep, isfinaltimestep
+    connectparameter, setleftoverparameters, getvariable, adder, MarginalModel, ConnectorComp, getindex,
+    getdataframe, components, variables, getvpd, unitcheck, plot, getindexcount,
+    getindexvalues, getindexlabels, delete!, get_unconnected_parameters, Timestep, isfirsttimestep,
+    isfinaltimestep, TimestepVector, TimestepMatrix, hasvalue
```

Comment:
Do we need to export ``Timestep``, ``TimestepVector`` and ``TimestepMatrix``? Users shouldn't really construct these types themselves, right?